ui/src/app/store/zone/constants.ts

```diff
@@ -0,0 +1,26 @@
+/* The following constants should be made generic, usable across models. */
+export const ACTION_STATUS = {
+  failed: "failed",
+  processing: "processing",
+  successful: "successful",
+} as const;
+/***************************/
+
+export const ZONE_PK = "id";
+
+export const ZONE_MODEL = "zone";
+
+export const ZONE_ACTIONS = {
+  cleanup: "cleanup",
+  create: "create",
+  delete: "delete",
+  fetch: "fetch",
+  update: "update",
+} as const;
```

Comment:
This is the single source of all the possible zone actions names as we refer to them in the codebase. This makes many things simpler as we should no longer need to map between the static and continuous forms of the action name, e.g. "update" => "updating".